app/models/miq_worker/container_common.rb

```diff
@@ -44,15 +44,18 @@ def default_image
     def resource_constraints
       mem_threshold = self.class.worker_settings[:memory_threshold]
       cpu_threshold = self.class.worker_settings[:cpu_threshold_percent]
+      mem_default   = self.class.worker_settings[:memory_minimum]
+      cpu_default   = self.class.worker_settings[:cpu_minimum_percent]
```

Comment:
Should the variables be something like `cpu_minimum` & `cpu_maximum` rather than `threshold` & `default`

Reply:
the threshold was introduced may 2020 - and in june it was changed to add percent.
wonder if we should call it limit so it is consistent with kubernetes?

the minimum is different from the memory minimum (which is a different concept) - so I renamed to request. (default is yet another setting which means the default limit)

I think it is ok being kubernets centric rather than systemd centric. And it would make more sense to the customer if we used names that were consistent with the kubernetes concepts

Reply:
Yeah, I know it's been around for a little while.  My concern is that `threshold` is ambiguous, it could be interpreted as min or max.